x-pack/solutions/security/plugins/security_solution/public/cloud_defend/index.ts

```diff
@@ -6,11 +6,16 @@
  */
 
 import type { SecuritySubPlugin } from '../app/types';
+import { routes } from './routes';
 
 export class CloudDefend {
   public setup() {}
 
-  public start(): SecuritySubPlugin {
-    return { routes: [] };
+  public start(isServerless?: boolean): SecuritySubPlugin {
+    if (isServerless ?? false) {
```

Comment:
nit: I think initializing with false might be easier to understand

```suggestion
  public start(isServerless: boolean = false): SecuritySubPlugin {
    if (isServerless) {
```

Reply:
it would be also good to write a comment  on why it's hidden on serverless